Dockerfile

```
@@ -1,6 +1,6 @@
 ### This is a generated file from Dockerfile.in ###
 #@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_8_golang_1.16)
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.8 AS builder
```

I think you need to update the "follow_tag" line as well. For the loki build we switched to "4.10" in the base image. The image you are now referring to does not seem to exist, so maybe 4.10 is the first version with Go 1.17.

```suggestion
#@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_8_golang_1.17)
FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 AS builder
```

The suggestion display seems wonky (one line too much), don't know if that UI feature will work here...

---

`go fmt` does not seem to do anything for me, so I manually fixed the `follow_tag`